### ISSUE TITLE
Feture Recrest

### ISSUE THREAD

**Issue by Scratchinator** [PascalCorpsman/mini_projects#1]
Im Lan Chat kommt eine naricht an und dan macht es bing, ist es möglich dieses bing auch durch andere töne ersezbat zu machen. Ich denke da an eine Tabele oder so etwas in den Einstelungen.

**Comment by PascalCorpsman** [comment]
Ja, der Ton der abgespielt wird ist die "nm.wav" diese kann beliebig ausgetauscht werden, oder hättest du gerne eine Auswahl mehrerer Töne, ggf auch unterschiedliche je nach Chat ?

**Comment by Scratchinator** [comment]
ja genau soetwas habe ich mir vorgestelt

**Comment by PascalCorpsman** [comment]
won't do, as it is not needed anymore
